src/components/ToDoComponent/TodoComponent.tsx

```diff
@@ -1,25 +1,26 @@
 import { Text, TouchableOpacity, View, StyleSheet } from 'react-native';
 import React, { useEffect, useState } from 'react';
 import Card from './Card';
-import data from './Data';
 import { TodoStyles } from './Styles/TodoStyles';
 import Task from './taskType';
+import Data from './Data';
 
 const TodoComponent = ({ navigationProp }) => {
   const [tasks, setTasks] = useState<Task[]>([]);
   const [disabled, setDisabled] = useState<boolean>(false);
   const [changed, setChanged] = useState<boolean>(false);
 
   useEffect(() => {
-    getTodos()
+    getTodos();
   }, [changed, tasks]);
 
-  const getTodos = async() => {
-    const todos = await fetch('https://backend-goals-production.up.railway.app/goal/');
+  const getTodos = async () => {
+    const todos = await fetch(
+      'https://backend-goals-production.up.railway.app/goal/',
```

Comment:
Can you please create a constant for this URL?

Reply:
moved